box: allow to access to the granted role in _vuser space (#10747)
tarantool/tarantool: changelogs/unreleased/gh-10057-box-schema-user-info.md

## bugfix/box

> * Fixed a bug when the `box.schema.user.info()` function could fail if a current
>   user is granted with a non-'public' role (gh-10057).

Reviewer: ```suggestion
* Fixed a bug when the `box.schema.user.info()` function could fail if a current
  user was granted with a non-`public` role (gh-10057).
```
